src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx

```diff
@@ -135,6 +136,27 @@ export default class ResultTable extends React.Component {
     return widths;
   };
 
+  getRowHeight = (rowIndex, columnsToCheck) => {
+    const {data: {data}} = this.props;
+
+    if (rowIndex === 0) {
+      return TABLE_ROW_HEIGHT_WITH_BORDER;
+    }
+
+    const row = data[rowIndex - 1]; // -1 offset due to header row
+    const colWidths = columnsToCheck.map(col => {
+      return this.measureText(getDisplayText(row[col]), false);
+    });
+    const maxColWidth = Math.max(...colWidths, 0);
+
+    const rows = Math.max(
+      Math.min(Math.ceil(maxColWidth / (MAX_COL_WIDTH - CELL_PADDING)), 3),
```

Comment:
Can you add a comment for what this calc is for?

Reply:
added